docs/.vuepress/tools/jsdoc-convert/integrations/jsdoc-to-markdown/index.mjs

```diff
@@ -0,0 +1,24 @@
+import jsdoc2md from 'jsdoc-to-markdown';
+import dmd from 'dmd';
+import path from 'path';
+import {fileURLToPath} from "url";
```

Comment:
This should be caught by the ESlint. Can you update the npm scripts and include the `.mjs` files in the lining process?

```
"docs:lint": "eslint --ext .js,.vue,.mjs .vuepress next",
"docs:lint:fix": "eslint --ext .js,.vue,.mjs .vuepress next --fix",
```

Reply:
Good catch, done bc8d959